Makefile

```diff
@@ -268,20 +270,12 @@ endif
 
 .PHONY: operator-sdk
 OPERATOR_SDK = ./bin/operator-sdk
-operator-sdk: ## Download operator-sdk locally if necessary.
-ifeq (,$(wildcard $(OPERATOR_SDK)))
```

Comment:
- better have this one atleast or-else sdk is downloaded everytime the target is called
- the downside will be a mismatch version, however that's better than downloading everytime I believe.

Reply:
The intention is to use the same version everytime as we are seeing different behaviour with different versions.

Reply:
yes, if the frequency of calling this target as a sub-target is less I'd also vote for this PR.